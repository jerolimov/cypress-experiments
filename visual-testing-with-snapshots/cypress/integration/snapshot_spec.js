describe('My first test', () => {
  it('visits the kitchen sink', () => {
    cy.visit('/')
    cy.contains('type').click()
    cy.url().should('include', '/commands/actions')
    cy.get('.action-email').type('fake@email.com')

    cy.get('.action-email').toMatchSnapshot();

    cy.get('.action-email').toMatchImageSnapshot();

    cy.document().toMatchImageSnapshot();
  })
})
