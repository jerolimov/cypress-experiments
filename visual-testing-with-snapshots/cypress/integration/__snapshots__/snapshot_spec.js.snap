exports[`My first test > visits the kitchen sink #0`] = `
<input
  type="email"
  class="form-control action-email"
  id="email1"
  placeholder="Email"
/>
`;
